File: phase-1-guide-nyc-master/Phase1-week2/orange-tree-1-just-oranges-challenge-master/spec/orange_spec.rb
require_relative "../orange"

describe Orange do
  let(:orange) { Orange.new }

  it "has a diameter between 2.5 and 3.2" do
    desired_diameter_range = (2.5..3.2)
    expect(desired_diameter_range).to cover orange.diameter
  end
end

File: phase-2-guide-nyc-for-unified.zip (Unzipped Files)/Phase2-week2/oojs-orange-tree-challenge-master/spec/orange_spec.js
describe("an orange", function() {
  var orange;

  beforeEach(function() {
    orange = new Orange();
  });

  it("has a diameter between 2.5 and 3.2", function() {
    expect(orange.diameter).not.toBeLessThan(2.5);
    expect(orange.diameter).not.toBeGreaterThan(3.2);
  });
});
